spec/commands/thredded/autofollow_users_spec.rb

```diff
@@ -71,7 +71,7 @@ module Thredded
     end
 
     context 'with thredded_user_preferences.auto_follow_topics default true' do
-      around(:all) do |ex|
+      around do |ex|
```

Comment:
If we add more tests, we should use a `before(:all)` and `after(:all)` hook to minimize the number of schema changes. Since it's just the one test, it doesn't matter here.